choco/nunit-console-with-extensions.nuspec

```diff
@@ -33,5 +33,15 @@
 	<language>en-US</language>
     <tags>nunit console runner test testing tdd</tags>
     <copyright>Copyright (c) 2017 Charlie Poole, Rob Prouse</copyright>
+    <dependencies>
+      <group>
+          <dependency id="nunit-console-runner" version="$version$" />
+          <dependency id="nunit-extension-nunit-project-loader" version="3.5.0" />
+          <dependency id="nunit-extension-vs-project-loader" version="3.5.0" />
+          <dependency id="nunit-extension-nunit-v2-result-writer" version="3.5.0" />
+          <dependency id="nunit-extension-nunit-v2-driver" version="3.6.0" />
+        <dependency id="nunit-extension-teamcity-event-listener" version="1.0.2" />
```

Comment:
Argh! Never mind! Yes, now I see what you mean. I released new versions of several of them just recently - in fact you reviewed them. But the console / engine project has not accepted those new releases to be bundled yet. I'm assuming that's a decision Rob makes when he releases.

It's a bit academic when the same people are in both roles, but my thinking is that the extension maintainer releases whenever he wants and the console maintainer decides what releases are well tested enough to bundle. We still don't have a new release of teamcity, by the way. It's in process.